### PR TITLE
Remodel the projector

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
Waiting for https://github.com/OpenSlides/openslides-backend/issues/462

@ostcar You did not do anything with the projector yet, right? So we must not wait for changes from you?

**Comment by ostcar** [comment]
> @ostcar You did not do anything with the projector yet, right? So we must not wait for changes from you?

I did not start yet. You don't have to wait

File: docs/models.yml [hunk 1]
```diff
@@ -1044,11 +1057,7 @@ agenda_item:
     equal_fields: meeting_id
   projection_ids:
     type: relation-list
-    to: projection/element_id
-    equal_fields: meeting_id
-  current_projector_ids:
-    type: relation-list
-    to: projector/current_element_ids
+    to: projection/content_object_id
```
Comment (jsangmeister) [inline docs/models.yml:1060]: Why did you remove the `equal_fields`? If this was by mistake, please add back (same for all other places below).

Comment (FinnStutzenstein) [reply docs/models.yml:1060]: It is in the line below your comment.

**Comment by FinnStutzenstein** [comment]
@jsangmeister If there are missing changes, please add them later.

File: docs/models.yml (hunk 1 above)
Comment (jsangmeister) [reply docs/models.yml:1060]: Oops, my bad
